src/client/ducks/repositories/selectors/index.js

```diff
@@ -72,3 +72,13 @@ export const getRepositoryId = createSelector(
     _.findKey(entities, (v) => (v.fullName === repoName))
 )
 
+export const getChangelogFetchStatus = statusFetchFactory(types.FETCH_CHANGELOG)
+export const getChangelog = (state: Object, props: Object) => {
+  const changesets = _.get(state, ['entities', 'changesets'], [])[undefined]
```

Comment:
it feels a bit weird that I have to access the data using `[undefined]`. There must be a better way, but right now I don't know how to correct it